.github/workflows/release.yml

```diff
@@ -0,0 +1,107 @@
+name: Release
+
+permissions:
+  contents: write
+  pull-requests: write
+
+env:
+  PANDOC_VERSION: 3.1.1.0
+
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
Is there no merged type, do we actually have to check it later in the action?

Reply:
jup ...
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request